data.sql

```diff
@@ -1,5 +1,12 @@
 /* Populate database with sample data. */
 
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES 
+   ('Agumon', '2020-02-03', 0, true, 10.23),
+  ('Gabumon', '2018-11-15', 2, true, 8),
+  ('Pikachu', '2021-01-07', 1, false, 15.04),
+  ('Devimon', '2017-05-12', 5, true, 11);
+
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
```

Comment:
- [ ] [Optional] You've made good progress, but I noticed that there's some unnecessary data in your project. To keep your project clean and focused on the requirements, I recommend removing this irrelevant data. This will help streamline your work and make your project more organized. Keep up the good work! 👍🧹